utilities/pipelines/staticValidation/compliance/module.tests.ps1

```diff
@@ -86,27 +85,30 @@ Describe 'File/folder tests' -Tag 'Modules' {
                     isTopLevelModule                   = ($resourceTypeIdentifier -split '[\/|\\]').Count -eq 2
                     childModuleAllowedList             = $childModuleAllowedList
                     childModuleAllowedListRelativePath = $childModuleAllowedListRelativePath
+                    versionFileExists                  = Test-Path (Join-Path $moduleFolderPath 'version.json')
+                    isMultiScopeModule                 = (Split-Path $moduleFolderPath -Leaf) -match '\/(rg|sub|mg)\-scope$'
+                    hasMultiScopeChildModules          = ((Get-ChildItem -Directory -Path $moduleFolderPath) | Where-Object { $_.FullName -match '\/(rg|sub|mg)\-scope$' }).Count -gt 0
```

Comment:
If I get it correctly with `isMultiScopeModule` we mean if this is a child module of a multi scope module, right?
In that case I'd suggest to rename it. Personally with `isMultiScopeModule` I would take it as the top level module instead, while that is, still if I got it correctly, the meaning of `hasMultiScopeChildModules`.

What about
```suggestion
                    isMultiScopeChildModule            = (Split-Path $moduleFolderPath -Leaf) -match '\/(rg|sub|mg)\-scope$'
                    isMultiScopeParentModule           = ((Get-ChildItem -Directory -Path $moduleFolderPath) | Where-Object { $_.FullName -match '\/(rg|sub|mg)\-scope$' }).Count -gt 0
```

Reply:
done